src/managed_scip.jl

```diff
@@ -276,3 +276,50 @@ function add_indicator_constraint(mscip::ManagedSCIP, y, x, a, rhs)
     @SC SCIPaddCons(mscip, cons__[])
     return store_cons!(mscip, cons__)
 end
+
+# Transform SCIP C function name as follows:
+# 1. Remove leading "SCIP" part (drop the first four characters).
+# 2. Convert camel case to snake case.
+# For example, `SCIPprintStatusStatistics` becomes `print_status_statistics`.
+const STATISTICS_FUNCS = map(x -> Symbol(camel_case_to_snake_case(string(x)[5 : end])), SCIP_STATISTICS_FUNCS)
+
+for (scip_statistics_func, statistics_func) in zip(SCIP_STATISTICS_FUNCS, STATISTICS_FUNCS)
```

Comment:
shouldn't this part also be a macro?

Reply:
This is how it's done in Base, e.g. https://github.com/JuliaLang/julia/blob/f0ffb29feb0499ce49d621acfa8a2ce192e8dba0/base/operators.jl#L524-L534.

Reply:
fair point